app/assets/stylesheets/custom.scss

```
@@ -141,3 +141,55 @@ input {
     border-bottom: 1px solid $gray-lighter;
   }
 }
+
+aside {
+  textarea {
+    height       : 100px;
```

```suggestion
    height: 100px;
```

Bỏ kiểu indent này đi nhé. Như này thôi